README.md

```diff
@@ -43,6 +43,30 @@ To keep the documentation website fast and responsive, we have a few recommendat
   Usually, you can reduce file size significantly by reducing the number of colors to a palette of 256 colours: `gifsicle -O3 --colors 256 -i input.gif -o output.gif`
 - **Use them sparingly**: Only use animated GIFs when they add significant value to the documentation. If a static image can convey the same information, prefer the image.
 
+### Inline images
+
+Regular markdown images work well for things like screenshots, but when documenting e.g. UIs, and you want to focus on a particular component on the screen, this might not be ideal.
+You have to explain for example "the button to the far right, second from the top", and this might not even be true on all devices and layouts.
+In situations like this, it can be better to make a small image of just the component, a button for example, and then use that inline in the text.
+
+To achieve this, a small Vue component called `InlineImage` has been created.
+Using it is outside "regular Markdown syntax", which is why it is explained here.
+Here is an example of the syntax:
+
+```vue
+<InlineImage alt="Select button" height="1.2em" :src="require('./images/select-button.png')" />
```

Comment:
Wouldn't it be possible to allow passing the path as src and then passing the src path into require inside the component? That would make its usage easier.

Reply:
I just tried to pass it along as a string and "transform it" inside the component, and this give me the following build error:
```
[Vue warn]: Error in render: "Error: Cannot find module './images/duplicate-rule-button.png'"
```
So, when it's done this way, the image "doesn't become a module" and thus isn't managed by the build and end up not existing in the right location.

I'm sure there's a way to do this, it's just way beyond my Vue knowledge to figure out how it would have to be. It's not *that* difficult to wrap it in `require()`, and as a bonus, the build fails if the image isn't there.

Reply:
Here is some more info on the subject, it seems to work fine if you "hard code" the image link in the component, but that kind of defeats the whole purpose of this component:

https://stackoverflow.com/questions/68450660/linking-to-image-from-cutom-component-in-vuepress
https://stackoverflow.com/a/45116994

Reply:
Okay, thanks for looking into this. Fine then.